pkg/common/pgedge-rag-server.yaml

```
@@ -0,0 +1,63 @@
+    description: "Search our documentation"
+    database:
+      host: "localhost"
+      port: 5432
+      database: "ragdb"
+      username: "docuser"
+      password: "your_secure_password"
+      ssl_mode: "prefer"
+      # hosts:
+      #     - host: "postgres-n1"
+      #       port: 5432
+      #     - host: "postgres-n2"
```

YAMLlint (1.37.1)

[error] duplication of key "      #     - host" in mapping

(key-duplicates)